### PR TITLE
#3875 ohara-kafka should explicitly require guava dependency

### PR DESCRIPTION
**Pull request by chia7712**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #3875)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by jackyoh** [comment]
retry connector

File: ohara-kafka/src/main/java/com/island/ohara/kafka/TopicAdmin.java [hunk 1]
```diff
@@ -333,7 +332,8 @@ else if (current.numberOfPartitions() == numberOfPartitions)
                     CompletableFuture<Void> f = new CompletableFuture<>();
                     admin
                         .createPartitions(
-                            ImmutableMap.of(name, NewPartitions.increaseTo(numberOfPartitions)))
+                            Collections.singletonMap(
+                                name, NewPartitions.increaseTo(numberOfPartitions)))
```
Comment (jackyoh) [inline ohara-kafka/src/main/java/com/island/ohara/kafka/TopicAdmin.java]: 不用 ImmutableMap 的原因是效能的考量還是因為 dependency 的原因？

Comment (chia7712) [reply ohara-kafka/src/main/java/com/island/ohara/kafka/TopicAdmin.java]: It would be better to adopt standard library than 3rd :)

Comment (jackyoh) [reply ohara-kafka/src/main/java/com/island/ohara/kafka/TopicAdmin.java]: Ok, 了解。Thank you.

**Pull request merged**